web/src/transmission.js

```diff
@@ -962,22 +985,27 @@ TODO: fix this when notifications get fixed
     const trackers = this._getTrackerCounts();
     const sitenames = Object.keys(trackers).sort();
 
-    // build the new html
-    let string = '';
-    string += this.filterTracker
-      ? '<option value="all">All</option>'
-      : '<option value="all" selected="selected">All</option>';
-    for (const sitename of sitenames) {
-      string += `<option value="${sitename}"`;
-      if (sitename === this.filterTracker) {
-        string += ' selected="selected"';
+    // Update select box only when list of trackers has changed
```

Comment:
This is tricky because of replacing trackers. To be sure you'd have to compare the old and new list of trackers.

Reply:
I'm not sure I'm seeing the problem, can you please illustrate the problem example? There may be reading comprehension problem in these codes, it got me at constant declaration for `sitenames`.

Reply:
My bad. I was probably reading diagonally. You ARE comparing the new with the old.

Reply:
You can make a change suggestion to switch them around and I'll accept it.